pkg/detectors/sentrytoken/sentrytoken.go

```diff
@@ -76,54 +87,59 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 	return results, nil
 }
 
-type Response []Project
-
-type Project struct {
-	Organization Organization `json:"organization"`
-}
-
-type Organization struct {
-	ID   string `json:"id"`
-	Name string `json:"name"`
-}
-
-func verifyToken(ctx context.Context, client *http.Client, token string) (bool, error) {
-	req, err := http.NewRequestWithContext(ctx, http.MethodGet, "https://sentry.io/api/0/projects/", nil)
+func verifyToken(ctx context.Context, client *http.Client, token string) (map[string]string, bool, error) {
+	// api docs: https://docs.sentry.io/api/organizations/
+	// this api will return 200 for user auth tokens with scope of org:<>
+	req, err := http.NewRequestWithContext(ctx, http.MethodGet, "https://sentry.io/api/0/organizations/", nil)
 	if err != nil {
-		return false, err
+		return nil, false, err
 	}
 	req.Header.Add("Authorization", fmt.Sprintf("Bearer %s", token))
 
-	res, err := client.Do(req)
+	resp, err := client.Do(req)
 	if err != nil {
-		return false, err
+		return nil, false, err
 	}
-	defer res.Body.Close()
+	defer func() {
+		_, _ = io.Copy(io.Discard, resp.Body)
+		_ = resp.Body.Close()
+	}()
 
-	var isVerified bool
-	switch res.StatusCode {
-	case http.StatusOK, http.StatusForbidden:
-		isVerified = true
-	case http.StatusUnauthorized:
-		return false, errUnauthorized
-	default:
-		return false, fmt.Errorf("unexpected HTTP response status %d", res.StatusCode)
+	bytes, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return nil, false, err
```

Comment:
This is unnecessary, you can read the body directly with `json.NewDecoder(res.Body).Decode(...)` instead of `json.Unmarshal(...)`.

Reply:
> How common is it for a token to have `org:` scope versus `project:` scope?  

It depends on the task the user is creating the token for. When generating a token, users have the option to add any scope they need, with at least one scope being mandatory. The reason for using the organization API in this case is that we're trying to retrieve some organization-level data for `extraData`.